_posts/2022-11-14-release-2022.11.14.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Release Day 2022.11.14"
+description: "Today is release day!"
+published: 2022-11-14
+layout: post
+thumb: https://raw.githubusercontent.com/bottlesdevs/data/main/releases/2022.10.14/release-day.png
+background: https://raw.githubusercontent.com/bottlesdevs/data/main/backgrounds/2022.10.14.png
+published: true
+comments: true
+is_release_day: true
+is_release_day_color: '#1699e7'
+user_nick: mirkobrombin
```

Comment:
not this time

Reply:
No worries. I'm reviewing right now and adding corrections.